Header header

string[] object_name
string[] object_color
geometry_msgs/Point object_position
geometry_msgs/Quaternion object_orientation
geometry_msgs/Vector3 object_scale
---
int64 w_flag
